src/vendoring/tasks/vendor.py

```diff
@@ -59,6 +60,11 @@ def rewrite_file_imports(
         text = re.sub(pattern, substitution, text)
 
     for lib in vendored_libs:
+        # Error on "import lib.module": this cannot be rewritten
+        match = re.search(rf"(\n\s*|^)(import {lib}\.\S*)", text)
```

Comment:
This also matches
```python
import X.Y as Z
```
However this can be rewritten to `import vendor.X.Y as Z`, see #18